code/modules/antagonists/nukeop/equipment/nuclear_challenge.dm

```diff
@@ -61,7 +61,37 @@ GLOBAL_LIST_EMPTY(jam_on_wardec)
 	for(var/obj/machinery/computer/camera_advanced/shuttle_docker/D in GLOB.jam_on_wardec)
 		D.jammed = TRUE
 
-	new uplink_type(get_turf(user), user.key, CHALLENGE_TELECRYSTALS)
+	var/list/orphans = list()
+	var/list/uplinks = list()
+
+	for (var/datum/mind/M in get_antag_minds(/datum/antagonist/nukeop))
+		var/datum/component/uplink/uplink = M.find_syndicate_uplink()
+		if (!uplink)
+			orphans += M.current
+			continue
+		uplinks += uplink
+
+
+	var/tc_to_distribute = CHALLENGE_TELECRYSTALS
+	var/tc_per_nukie = round((tc_to_distribute / length(get_antag_minds(/datum/antagonist/nukeop))))
```

Comment:
should this check for nukie uplinks instead of antag status? i think reinforcements start without an uplink and may count stuff like syndi borgs